dpos/state/arbitrators.go

```diff
@@ -1019,31 +1101,37 @@ func NewArbitrators(chainParams *config.Params, bestHeight func() uint32,
 	}
 
 	a := &arbitrators{
-		chainParams:                     chainParams,
-		bestHeight:                      bestHeight,
-		bestBlock:                       bestBlock,
-		getBlockByHeight:                getBlockByHeight,
-		currentOwnerProgramHashes:       originArbitersProgramHashes,
-		nextArbitrators:                 originArbiters,
-		nextCandidates:                  make([][]byte, 0),
-		nextOwnerProgramHashes:          originArbitersProgramHashes,
-		nextCandidateOwnerProgramHashes: make([]*common.Uint168, 0),
-		crcArbiters:                     crcArbiters,
-		crcArbitratorsNodePublicKey:     crcNodeMap,
-		crcArbitratorsProgramHashes:     crcArbitratorsProgramHashes,
-		accumulativeReward:              common.Fixed64(0),
-		finalRoundChange:                common.Fixed64(0),
-		arbitersRoundReward:             nil,
-		illegalBlocksPayloadHashes:      make(map[common.Uint256]interface{}),
-		currentTotalVotesInRound:        0,
-		nextTotalVotesInRound:           0,
-		currentOwnerVotesInRound:        make(map[common.Uint168]common.Fixed64),
-		nextOwnerVotesInRound:           make(map[common.Uint168]common.Fixed64),
-		snapshots:                       make(map[uint32][]*KeyFrame),
-		snapshotKeysDesc:                make([]uint32, 0),
+		chainParams:                 chainParams,
+		store:                       store,
+		bestHeight:                  bestHeight,
+		bestBlock:                   bestBlock,
+		getBlockByHeight:            getBlockByHeight,
+		nextArbitrators:             originArbiters,
+		nextCandidates:              make([][]byte, 0),
+		crcArbiters:                 crcArbiters,
+		crcArbitratorsNodePublicKey: crcNodeMap,
+		crcArbitratorsProgramHashes: crcArbitratorsProgramHashes,
+		accumulativeReward:          common.Fixed64(0),
+		finalRoundChange:            common.Fixed64(0),
+		arbitersRoundReward:         nil,
+		illegalBlocksPayloadHashes:  make(map[common.Uint256]interface{}),
+		snapshots:                   make(map[uint32][]*KeyFrame),
+		snapshotKeysDesc:            make([]uint32, 0),
 		KeyFrame: &KeyFrame{
 			CurrentArbitrators: originArbiters,
 		},
+		CurrentReward: RewardData{
```

Comment:
Here can use a `NewRewardData` method instead of duplicate initializations.

Reply:
Fixed